crates/ruff_python_parser/src/parser/pattern.rs

```diff
@@ -554,15 +554,15 @@ impl<'src> Parser<'src> {
         cls: Pattern,
         start: TextSize,
     ) -> ast::PatternMatchClass {
+        let arguments_start = self.node_start();
+
         self.bump(TokenKind::Lpar);
 
         let mut patterns = vec![];
         let mut keywords = vec![];
         let mut has_seen_pattern = false;
         let mut has_seen_keyword_pattern = false;
 
-        let arguments_start = self.node_start();
-
```

Comment:
Silly mistake, the node should always start _before_ we move the parser any further ahead.